### PR TITLE
Added 2023-07-04 meeting notes

### LINKED ISSUE #3405: Node.js  Build WorkGroup Meeting 2023-07-04
## Time

**UTC Tue 04-Jul-2023 12:00 (12:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 04-Jul-2023 05:00 (05:00 AM) |
| US / Mountain | Tue 04-Jul-2023 06:00 (06:00 AM) |
| US / Central  | Tue 04-Jul-2023 07:00 (07:00 AM) |
| US / Eastern  | Tue 04-Jul-2023 08:00 (08:00 AM) |
| EU / Western  | Tue 04-Jul-2023 13:00 (01:00 PM) |
| EU / Central  | Tue 04-Jul-2023 14:00 (02:00 PM) |
| EU / Eastern  | Tue 04-Jul-2023 15:00 (03:00 PM) |
| Moscow        | Tue 04-Jul-2023 15:00 (03:00 PM) |
| Chennai       | Tue 04-Jul-2023 17:30 (05:30 PM) |
| Hangzhou      | Tue 04-Jul-2023 20:00 (08:00 PM) |
| Tokyo         | Tue 04-Jul-2023 21:00 (09:00 PM) |
| Sydney        | Tue 04-Jul-2023 22:00 (10:00 PM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Build%20WorkGroup+Meeting+2023-07-04&iso=20230704T1200
* or https://www.wolframalpha.com/input/?i=12PM+UTC%2C+Jul+04%2C+2023+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1l0rtHesVkymetXOXYitQJFDnFN6UPYNh-uehiSCgg2E/edit>

## Agenda

Extracted from **build-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/build

* DNS Wildcard problem?[#3404](https://github.com/nodejs/build/issues/3404)
* NearForm Benchmarking Servers [#3390](https://github.com/nodejs/build/issues/3390)
* Action required by Apple: Transition to the notarytool command-line utility [#3385](https://github.com/nodejs/build/issues/3385)
* Setup Terraform Cloud account [#3370](https://github.com/nodejs/build/issues/3370)
* Experiment with Node.js Website Traffic on Vercel [#3366](https://github.com/nodejs/build/issues/3366)
* Infrastructure for MacOS 12.x [#3240](https://github.com/nodejs/build/issues/3240)
* Access to Cloudflare [#3220](https://github.com/nodejs/build/issues/3220)
* Membership audit [#3144](https://github.com/nodejs/build/issues/3144)
* Support for llnode builds [#3017](https://github.com/nodejs/build/issues/3017)
* Interest in establishing a "Build Helper" role [#2550](https://github.com/nodejs/build/issues/2550)

## Invited

* Build team: @nodejs/build

## Notes

The agenda comes from issues labelled with `build-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: <https://zoom.us/j/715960833>
* For those who just want to watch: We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at <https://www.youtube.com/c/nodejs+foundation/live> when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
* youtube admin page: <https://www.youtube.com/my_live_events?filter=scheduled>

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure

### PR DESCRIPTION
**Pull request by UlisesGascon**
close #3405

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2023-07-04.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## Announcements
+
+* Extracted from **build-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/build
+
+* DNS Wildcard problem?[#3404](https://github.com/nodejs/build/issues/3404)
+  * Wildcard was implemented in the past and we don’t have the full context 
+  * This topic was added to the agenda for awareness
+  * @Richard is planning to remove the wildcard and add explicit DNS records, this should not be a breaking change (Cloudflare level change not Nginx)
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2023-07-04.md:28]: We should remove the `@` here (and everywhere else in these notes) unless we want to switch these mentions to use the GitHub handles.
```suggestion
  * Richard is planning to remove the wildcard and add explicit DNS records, this should not be a breaking change (Cloudflare level change not Nginx)
```

Comment (UlisesGascon) [reply doc/meetings/build-wg-meeting-2023-07-04.md]: Great catch!

Comment (targos) [reply doc/meetings/build-wg-meeting-2023-07-04.md]: There are many others

Comment (UlisesGascon) [reply doc/meetings/build-wg-meeting-2023-07-04.md]: I did a find/replace in 1f03163 👍

File: doc/meetings/build-wg-meeting-2023-07-04.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  * Ulises will lead this initiative with the support from Richard and Michaël
+  * Ulises will need to get an Apple Developer Account and might need access to certificates (TBC). The team seems comfortable with this idea, we will open an issue for the permission if needed
+* Terraform 
+  * Related: Setup Terraform Cloud account [#3370](https://github.com/nodejs/build/issues/3370)
+  * @Ulises has created a PR for supporting Terraform: https://github.com/nodejs/build/pull/3391
+  * @Ulises will re-sync the terraform files to match the current Cloudflare state
+  * @Richard / @Michaël will add the token missing for the PR in order to properly run the new Github actions pipeline
+  * Any change from terraform files won’t trigger any real change in Cloudflare as we are using read only tokens, we agree to keep this approach until we feel ready to enable write access.
+  * In order to prevent accidental deployments or similar, we agreed to update the logic in the pipeline and do `terraform apply` only when there is an event like new release/deployment created, @Ulises will update the logic
+  * @Michaël will add the Terraform Cloud Token to the secrets repo
```
[the remark this answers is not in the document]
Comment (UlisesGascon) [reply doc/meetings/build-wg-meeting-2023-07-04.md]: @targos the GitHub UI does not allowed me to commit the suggestion, so I added it on 9878e87

**Review by targos — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Pull request merged**